Traceback (most recent call last):
  File "/home/edu/anaconda3/lib/python3.7/site-packages/jupyter_cache/executors/utils.py", line 56, in single_nb_execution
    record_timing=False,
  File "/home/edu/anaconda3/lib/python3.7/site-packages/nbclient/client.py", line 1204, in execute
    return NotebookClient(nb=nb, resources=resources, km=km, **kwargs).execute()
  File "/home/edu/anaconda3/lib/python3.7/site-packages/nbclient/util.py", line 84, in wrapped
    return just_run(coro(*args, **kwargs))
  File "/home/edu/anaconda3/lib/python3.7/site-packages/nbclient/util.py", line 62, in just_run
    return loop.run_until_complete(coro)
  File "/home/edu/anaconda3/lib/python3.7/asyncio/base_events.py", line 587, in run_until_complete
    return future.result()
  File "/home/edu/anaconda3/lib/python3.7/site-packages/nbclient/client.py", line 664, in async_execute
    cell, index, execution_count=self.code_cells_executed + 1
  File "/home/edu/anaconda3/lib/python3.7/site-packages/nbclient/client.py", line 965, in async_execute_cell
    await self._check_raise_for_error(cell, cell_index, exec_reply)
  File "/home/edu/anaconda3/lib/python3.7/site-packages/nbclient/client.py", line 862, in _check_raise_for_error
    raise CellExecutionError.from_cell_and_msg(cell, exec_reply_content)
nbclient.exceptions.CellExecutionError: An error occurred while executing the following cell:
------------------
nombre_archivo_video = "polariscopiolineal.mkv" #escribir entre las comillas el nombre del archivo
#NO TOCAR
from IPython.display import Video
Video(nombre_archivo_video)
------------------

[0;31m---------------------------------------------------------------------------[0m
[0;31mValueError[0m                                Traceback (most recent call last)
[0;32m/tmp/ipykernel_20437/2856149666.py[0m in [0;36m<module>[0;34m[0m
[1;32m      2[0m [0;31m#NO TOCAR[0m[0;34m[0m[0;34m[0m[0;34m[0m[0m
[1;32m      3[0m [0;32mfrom[0m [0mIPython[0m[0;34m.[0m[0mdisplay[0m [0;32mimport[0m [0mVideo[0m[0;34m[0m[0;34m[0m[0m
[0;32m----> 4[0;31m [0mVideo[0m[0;34m([0m[0mnombre_archivo_video[0m[0;34m)[0m[0;34m[0m[0;34m[0m[0m
[0m
[0;32m~/anaconda3/lib/python3.7/site-packages/IPython/core/display.py[0m in [0;36m__init__[0;34m(self, data, url, filename, embed, mimetype, width, height, html_attributes)[0m
[1;32m   1408[0m                 [0;34m"Consider passing Video(url='...')"[0m[0;34m,[0m[0;34m[0m[0;34m[0m[0m
[1;32m   1409[0m             ])
[0;32m-> 1410[0;31m             [0;32mraise[0m [0mValueError[0m[0;34m([0m[0mmsg[0m[0;34m)[0m[0;34m[0m[0;34m[0m[0m
[0m[1;32m   1411[0m [0;34m[0m[0m
[1;32m   1412[0m         [0mself[0m[0;34m.[0m[0mmimetype[0m [0;34m=[0m [0mmimetype[0m[0;34m[0m[0;34m[0m[0m

[0;31mValueError[0m: To embed videos, you must pass embed=True (this may make your notebook files huge)
Consider passing Video(url='...')
ValueError: To embed videos, you must pass embed=True (this may make your notebook files huge)
Consider passing Video(url='...')

